logging/test/version_test.dart

```diff
@@ -14,6 +14,5 @@ void main() {
           yaml.loadYaml(File('pubspec.yaml').readAsStringSync());
       expect(sdkVersion, pubspec['version']);
     },
-    skip: 'works locally but not on CI',
```

Comment:
maybe remove this whole file completely? It's just a duplicate of the other one.

Reply:
They are under different packages, and each package has its own version, so if https://github.com/getsentry/sentry-dart/blob/main/scripts/bump-version.sh fails, this test would fail.